src/adapters/store.js

```diff
@@ -1,93 +1,111 @@
 import Error from '../adapters/error';
 import { version } from '../../config/constants.yml';
 import { findIndexIgnoreCase } from '../libs/string';
-import LocalStore from '../libs/local_store';
 import { getKey } from 'src/libs/pois';
 import { fire } from 'src/libs/customEvents';
+import { isPoiCompliantKey } from 'src/libs/pois';
 
-export default class Store {
-  constructor() {
-    // get store from window if already initialized
-    if (window.__store) {
-      return window.__store;
+class Store {
```

Comment:
:+1
What do you think about even removing the class and exporting only functions ?

Reply:
Yep, even simpler than a singleton, but I wanted to keep this PR with minimal changes outside the store file itself.
Do you want me to make the change here?

Reply:
would be great !

Reply:
Sure, doing it!